volunteer/static/css/volunteer.css

```diff
@@ -26,3 +26,18 @@ div.role h4 a.role-quicklink {
 div.role h4:hover a.role-quicklink {
     display: inline;
 }
+
+
+/*
+ * Background and container backgrounds
+ * to compliment bootstra-black.min.css
+ */
+body {   
+    background: #000 url("http://apogaea.com/wp-content/uploads/2015/01/Apogaea_web_2015_flat03.jpg") no-repeat center 40px;
```

Comment:
Will you pull this image into the project and change this path to `../img/name-of-image-file.jpg`